src/dnx.coreclr.unix/stdafx.h

```diff
@@ -0,0 +1,13 @@
+#include <dlfcn.h>
+#include <limits.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <string>
+#include <unistd.h>
+
+typedef char* LPWSTR;
```

Comment:
I agree but this is a point in time issue.  This is needed because I wanted to share as much of tpa.[h|cpp] between windows and *nix.  I will fix this later by moving to TCHAR like elsewhere.